content/fr/agent/kubernetes/daemonset_setup.md

```diff
@@ -15,7 +15,7 @@ aliases:
   - /agent/kubernetes/apm
   - /integrations/faq/using-rbac-permission-with-your-kubernetes-integration
 ---
-Tirez profit des DaemonSets pour déployer l'Agent Datadog sur l'ensemble de vos nœuds (ou sur un nœud donné grâce [aux nodeSelectors][1]). 
+Tirez profit des DaemonSets pour déployer l'Agent Datadog sur l'ensemble de vos nœuds (ou sur un nœud donné grâce [aux nodeSelectors][1]).
```

Comment:
The french documentation should not be updated via PR. We have an translation service for this.

Reply:
Indeed @ruthnaebeck is right. We can merge this PR anyway, it's just the the FR changes are going to be overridden by the next translation PR